### PR TITLE
style: change parameter style to avoid_final_parameters

### PR DESCRIPTION
**Pull request by Leptopoda**
fixes: #1245

This does not migrate all of our packages and I don't see the need to do this any time soon.
For the bigger ones like dynamite or the framework we could also just keep it for ever.

### CODE REVIEW

**Comment by provokateurin** [comment]
I'd prefer a single style for all packages. Either remove them everywhere or keep them everywhere.

**Comment by Leptopoda** [comment]
Can we still do this incrementally?

**Comment by provokateurin** [comment]
Sure

File: packages/nextcloud/build.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ targets:
             - public_member_api_docs
             - unreachable_switch_case
             - camel_case_extensions
+            - no_leading_underscores_for_local_identifiers
```
Comment (provokateurin) [inline packages/nextcloud/build.yaml:15]: Please introduce this in a separate PR as it has nothing to do with the final parameters.

**Review by provokateurin — CHANGES_REQUESTED** [review]
Please only change the rules for the packages that you are migrating in this PR instead of changing the default and overriding it in all other packages.

**Comment by Leptopoda** [comment]
> Please only change the rules for the packages that you are migrating in this PR instead of changing the default and overriding it in all other packages.

I disagree. While I will try to tackle this in the coming days the individual migrations (especially the framework) will take some time.

If we where to introduce a new package in that time it should automatically have the new rules.

**Comment by provokateurin** [comment]
Ok I get that point. Then please add a TODO comment everywhere where the rules are manually changed.

**Review by provokateurin — APPROVED** [review]

**Pull request merged**